hivemind/client/dht_ops.py

```diff
@@ -0,0 +1,67 @@
+from functools import partial
```

Comment:
Note: this file contains a subset of DHT methods from 0.9.5. These methods now work on top of DHT.run_coroutine, but all algorithms are exactly the same.